scripts/seed-database/write.sql

```diff
@@ -0,0 +1,135 @@
+BEGIN;
+
+  -- upsert users
+  CREATE TEMPORARY TABLE sync_users (
+    id integer,
+    first_name text,
+    last_name text,
+    email text,
+    is_admin boolean,
+    created_at timestamptz,
+    updated_at timestamptz
+  );
+
+  \copy sync_users FROM '/tmp/users.csv'  WITH (FORMAT csv, DELIMITER ';');
+
+  INSERT INTO users (
+    id,
+    first_name,
+    last_name,
+    email,
+    is_admin
+  )
+  SELECT
+    id,
+    first_name,
+    last_name,
+    email,
+    is_admin
+  FROM sync_users
+  ON CONFLICT (id) DO NOTHING;
+
+  -- upsert teams
+  CREATE TEMPORARY TABLE sync_teams (
+    id integer,
+    name text,
+    slug text,
+    theme jsonb,
+    created_at timestamptz,
+    updated_at timestamptz,
+    settings jsonb,
+    notify_personalisation jsonb,
+    domain text,
+    submission_email text
+  );
+
+  \copy sync_teams FROM '/tmp/teams.csv' WITH (FORMAT csv, DELIMITER ';');
+
+  INSERT INTO teams (
+    id,
+    name,
+    slug,
+    theme,
+    settings,
+    notify_personalisation,
+    domain,
+    submission_email
```

Comment:
follow-on comment from above: I think we want to omit `domain` and `submission_email` from this list of fields!